fedmsg_migration_tools/verify_missing.py

```diff
@@ -0,0 +1,168 @@
+import json
+import logging
+from datetime import datetime, timedelta
+
+from fedora_messaging import config
+from fedora_messaging.twisted.producer import MessageProducer
+
+from twisted.internet import defer, reactor, task
+from twisted.application import service
+from twisted.logger import Logger
+from txzmq import (
+    ZmqEndpoint, ZmqEndpointType, ZmqFactory, ZmqSubConnection,
+)
+
+
+_log = logging.getLogger(__name__)
+_txlog = Logger()
+
+
+class MasterService(service.MultiService):
+    """Multi service that stops the reactor with itself."""
+
+    def stopService(self):
+        d = super(MasterService, self).stopService()
+        d.addCallback(lambda _: reactor.stop())
+        return d
+
+
+class AmqpConsumer(service.Service):
+
+    QUEUE_NAME = "verify_missing"
+
+    def __init__(self, store):
+        self.store = store
+        self.producer = MessageProducer(
+            self.on_message, self._get_bindings())
+
+    def _get_bindings(self):
+        exchanges = [b["exchange"] for b in config.conf['amqp_to_zmq']['bindings']]
+        exchanges.append(config.conf['zmq_to_amqp']['exchange'])
```

Comment:
Do we want to document these settings in fedora-messaging or keep them private? If we keep them private we should still include a note somewhere non-user-facing.

Reply:
Those are the migration tools settings, so I think they should not go into fedora-messaging but stay with the `fedmsg_migration_tools` package.

Reply:
Okay. I was planning on adding configuration validation that would complain if there were unknown configuration keys (to save users from typos and to discourage using it as a dumping ground for their app's configuration), but adding an exception for fedmsg_migration_tools doesn't seem unreasonable.

Reply:
Hmm, if these keys are similar to an app's configuration, maybe the `fedmsg_migration_tools` code should load a separate configuration file. That would be an example for other apps using the fedora-messaging package.